### PR TITLE
Clean up dsse parser

### PR DESCRIPTION
**Pull request by naveensrinivasan**
- Some minor cleanup of dsse parser

Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Review by pxp928 — APPROVED** [review]
LGTM

File: pkg/ingestor/parser/dsse/parser_dsse.go [hunk 1]
```diff
@@ -42,17 +42,17 @@ func NewDSSEParser() common.DocumentParser {
 // Parse breaks out the document into the graph components
 func (d *dsseParser) Parse(ctx context.Context, doc *processor.Document) error {
 	d.doc = doc
-	err := d.getIdentity(ctx)
-	if err != nil {
-		return err
+
+	if err := d.getIdentity(ctx); err != nil {
+		return fmt.Errorf("getIdentity returned error: %v", err)
```
Comment (lumjjb) [inline pkg/ingestor/parser/dsse/parser_dsse.go:47]: %w instead of %v

Comment (naveensrinivasan) [reply pkg/ingestor/parser/dsse/parser_dsse.go:47]: Took care of it. Thanks

**Review by lumjjb — APPROVED** [review]
lgtm

**Pull request merged**